### DISCUSSION TITLE
Data transfer between systems Synology to TrueNas Scale.

### DISCUSSION THREAD

**Discussion by FloydKulis** [firefly-iii/firefly-iii#7573] [Firefly III]
How do I connect to the FireFly III application database on TrueNas Scale?
I need to move database from MeridaDB on Synology to FireFly III installed on TrueNas Scale.
Thank you for your help.

**Comment by JC5** [comment]
Are they both using Docker?

**Comment by FloydKulis** [comment]
Yes both are in Docker. The difference is that in Synology the database is elsewhere (at least I think so) and in TrueNas I don't know where the database is because the installation is automatic without configuration.
I didn't really think about it but I will upload all directories from Synology to TrueNAs, maybe it will work.

**Reply by JC5** [reply]
If they're both mysql, they'll both have mysqldump to import and export the database.

**Reply by FloydKulis** [reply]
In Synology, I have a database connection via PHPMyAddmin and I can export the database there. I can't connect to the TrueNas database yet.

**Reply by JC5** [reply]
Mmm sorry, I know very little about TrueNas myself.
